packages/react-lowcode/src/codegen/generation/generators/detail/grommet-detail-generator.ts

```diff
@@ -317,13 +320,13 @@ export default class GrommetDetailGenerator
       case PropertyType.string:
         assignment = factory.createPropertyAssignment(
           factory.createIdentifier(propertyName),
-          factory.createIdentifier("customer." + propertyName)
+          factory.createIdentifier(this._entityName.toLowerCase() + "." + propertyName)
         );
         break;
       case PropertyType.datetime:
         assignment = factory.createPropertyAssignment(
           factory.createIdentifier(propertyName),
-          factory.createIdentifier("customer." + propertyName)
+          factory.createIdentifier(this._entityName.toLowerCase() + "." + propertyName)
```

Comment:
wrap this please to something like `dataPropertyName = this._entityName.toLowerCase()`